### PR TITLE
Leaves, Breads and Pizzas re texture

### PR DESCRIPTION
**Pull request by BlueHero233**
To make leaves not identical and some other foodstuffs

### CODE REVIEW

**Review by boubou19 — APPROVED** [review]

**Comment by Ethryan** [comment]
Beautiful!

**Comment by LazyFleshWasTaken** [comment]
I love you

**Comment by leagris** [comment]
One issue though,with leaves/beries/fruits items. They are no longer like their block crop counterpart.
Would you edit the block texture for those so the leaves, fruits are same on the item and crop blocks?

File: src/main/resources/assets/gregtech/textures/items/gt.metaitem.02/502.png
Comment (leagris) [inline src/main/resources/assets/gregtech/textures/items/gt.metaitem.02/502.png:1]: Indigo leaves item
Deleted old
Added new
no longer look like the block crop texture 

Same issue with multiple leaves/fruit items

**Review by leagris — CHANGES_REQUESTED** [review]
Could you please try to also texture the corresponding blocks crop textures so they match their item counterpart?

Example with Indigo:

Indigo leaves item
Deleted old
Added new
no longer look like the block crop texture 

Same issue with multiple leaves/fruit items

File: src/main/resources/assets/gregtech/textures/items/gt.metaitem.02/502.png
Comment (BlueHero233) [reply src/main/resources/assets/gregtech/textures/items/gt.metaitem.02/502.png:1]: Added stages 4 and 3 for all modified crops

**Comment by BlueHero233** [comment]
> One issue though,with leaves/beries/fruits items. They are no longer like their block crop counterpart. Would you edit the block texture for those so the leaves, fruits are same on the item and crop blocks?

Done

**Review by leagris — APPROVED** [review]
Ok good. Thank you @BlueHero233 for all those nice civilized textures

**Pull request merged**